components/forms/w-affix.vue

```diff
@@ -16,9 +18,7 @@ const classBase = computed(() => (props.prefix ? prefix : suffix));
 
 const wrapperClass = computed(() => ({
```

Comment:
This would have been sooo much nicer. 😄 (But maybe not possible with via the linter 😅 )
```
const wrapperClass = computed(() => [
  classBase.value.wrapper,
  props.label ? classBase.value.wrapperWithLabel : classBase.value.wrapperWithIcon,
]);
```